hook!-full-game/src/Player/Player.tscn

```diff
@@ -34,8 +34,6 @@ shape = SubResource( 1 )
 position = Vector2( 0, -30 )
 
 [node name="LedgeDetector" parent="." instance=ExtResource( 3 )]
-active = true
```

Comment:
I'm not entirely sure why this is changing, but I don't see a particular problem with it, these are the default values in the `LedgeDetector` scene anyway.